mcu_janus.go

```diff
@@ -1063,7 +1063,11 @@ func (p *mcuJanusSubscriber) handleEvent(event *janus.EventMsg) {
 			log.Printf("Subscriber %d: associated room has been destroyed, closing", p.handleId)
 			go p.Close(ctx)
 		case "event":
-			// Ignore events like selected substream / temporal layer.
+			// Handle renegotiations, but ignore other events like selected
+			// substream / temporal layer.
+			if getPluginStringValue(event.Plugindata, pluginVideoRoom, "configured") == "ok" {
```

Comment:
This should check `event.Jsep` to make sure an offer has been received (`"type" = "offer" and "sdp" = "..."`) in case there are other events that can be sent by Janus. We can defer this to a follow-up PR.

Reply:
I have added a fixup for that. I tested it and as far as I can tell it works, but please check the syntax :-) Thanks!